cartographer/internal/mapping/global_trajectory_builder.h

```diff
@@ -59,17 +59,24 @@ class GlobalTrajectoryBuilder : public mapping::TrajectoryBuilderInterface {
       return;
     }
     std::unique_ptr<mapping::NodeId> node_id;
+    std::unique_ptr<InsertionResult> insertion_result;
     if (matching_result->insertion_result != nullptr) {
       node_id = ::cartographer::common::make_unique<mapping::NodeId>(
```

Comment:
Remove the `unique_ptr`which is no longer used?

Reply:
Done.